development/etiquette.md

```diff
@@ -0,0 +1,49 @@
+# Etiquette
+> Karen has an etiquette? Strange 🤔
+
+Listen up everyone, this will address the Karen Bot etiquette
+
+## Commit etiquette
+- Only push small commits (e.g if you changed 2 files, commit one file, commit the other and only <i>then</i> push) 🤏
+- Keep commit titles short and explain them in the description of the commit 
+- **DO NOT PUSH TO MASTER BRANCH!** Push to a new branch (e.g username-whatyouchanged and create a PR)
+
+## Pull Request etiquette
+So, this is the etiquette for Pull Requests. Pull Requests are a very useful thing for maintainers and it ensures that the correct people can review your code before it goes into production. That said, let's lay out the rules.
```

Comment:
Rather than "rules" I'd rephrase it to "conditions" or "conditions for contributing", sounds more friendly imo.

Reply:
I like "rules" more. "conditions" is too friendly for Karen Bot